charts/backend/templates/deployment.yaml

```
@@ -101,18 +101,38 @@ spec:
 {{- if ne (len .Values.extraVolumeMounts) 0 }}
 {{ toYaml .Values.extraVolumeMounts | indent 12 }}
 {{- end }}
+
+{{ if .Values.probes.backend }}
+          livenessProbe:
+            httpGet:
+              path: /status/health
+              port: http
+            initialDelaySeconds: {{ if .Values.probes.backend.livenessProbe.initialDelaySeconds }}{{ .Values.probes.backend.livenessProbe.initialDelaySeconds }}{{ else }}30{{ end }}
```

The values.yaml already provides us with default settings, so I think we can switch the if statements here.

```suggestion
            initialDelaySeconds: {{ .Values.probes.backend.livenessProbe.initialDelaySeconds }}
```

---

can we assume each organisation using this helm chart will update their values.yaml? If they don't it will result in a slight interruption.